### ISSUE TITLE
Modyfing a file deletes alternate data streams 

### ISSUE THREAD

**Issue by kjarex** [Komodo/KomodoEdit#337]
Let's say I have a file `file.txt` (like created by `ECHO 123 > file.txt`) And this file has an alternate data stream, let's say `file.txt:abc` (created by `ECHO 456 > file.txt:abc`).
If I modify `file.txt` with Komodo, the ADS `file.txt:abc` gets deleted (expected result would be, that the ADS still exists without any changes - this happens like that for example with MS Notepad or "Programmer's Notepad").

**Comment by geckotier** [comment]
May I ask why it is labeled as an enhancement and not as a bug?
Which criteria is the issue missing to be considered as a bug?

**Comment by cgchoffman** [comment]
Because it's not supported.  It will be an enhancement to accommodate a NTFS only attribute which will only service ~1/3 of the platforms we support.

**Comment by geckotier** [comment]
I totally agree and understand that this might affect a minority only (otherwise it would had been reported and fixed already long time ago). Therefor I also can understand if it doesn't get your highest priority - that's okay.

I wouldn't refer the "~1/3" as "only", ~1/3 is actually a sizeable amount. Anyway: popularity shouldn't be a requirement for an issue to be considered as a bug. Data loss should always be considered as a bug, even the worst kind of a bug.
If NTFS is not supported it is arguable if Windows should be called supported, since Windows requires NTFS since everything which came after XP. However, since this issue doesn't affect mounted NTFS files on MacOS (I haven't tested it on Linux) the intention to accommodate this attribute only while using Windows seems anyway kind of negligible.

**Comment by kjarex** [comment]
Are there any news about this "enhancement" being possibly seen as a bug?

**Comment by th3coop** [comment]
To clarify, I didn't say we don't support NTFS.  I meant we don't support the elusive and relatively unknown ADS.

You're right that losing data shouldn't be tolerated and thankfully it was a very easy fix.

I will provide a link to download a nightly build that will be available by 12 noon pst tomorrow.

**Comment by th3coop** [comment]
Nightly will be available here at 12 noon pst: http://downloads.activestate.com/Komodo/nightly/komodoide/

**Comment by th3coop** [comment]
As I had worried, that caused issues across the board.  I don't have time to look at another fix for this.  If someone comes up with something else, feel free to submit a PR.

**Comment by th3coop** [comment]
I've reverted the change and will delete the nightly.  For references, it caused: https://github.com/Komodo/KomodoEdit/issues/3865
